tests/data_structures/graph/test_graph.py

```diff
@@ -0,0 +1,16 @@
+from data_structures.graph.graph import AdjacencyList
+
+def test_graph():
+    '''Testing the creation and converting graph into adjacency list'''
+    al = AdjacencyList()
+    al.addEdge(0, 1)
+    al.addEdge(0, 4)
+    al.addEdge(4, 1)
+    al.addEdge(4, 3)
+    al.addEdge(1, 0)
+    al.addEdge(1, 4)
+    al.addEdge(1, 3)
+    al.addEdge(1, 2)
+    al.addEdge(2, 3)
+    al.addEdge(3, 4)
+    assert al.List == {0: [1, 4], 4: [1, 3], 1: [0, 4, 3, 2], 2: [3], 3: [4]}
```

Comment:
Every list in the dictionary should be sorted before comparison, since no specific order is required.